### DISCUSSION TITLE
Insert 0 for track no. less than 10 or 00 for track no. less than 100

### DISCUSSION THREAD

**Discussion by Greebo80** [audiamus/AaxAudioConverter#154] [Ideas]
First of all i love your program.

But now the question...
is there a possibility to insert automatically a 0 in front of the track numbers?
My car radio is messing around with file names like "1. dummy name" next track it would play would be "11. dummy name".
My only option is to insert a 0 in front of the track to keep the order.
With tracks from 100 I would even need a double 0...
Solve the problem currently with antrenamer...

**Comment by audiamus** [comment]
You are referring to track file names I guess. Leading zeros for track numbers should be there. But it may depend on your settings. Without explicit chapter names activated, chapters are simply numbered in increasing order, with leading zeros. However, if the explicit chapter names are used, the chapter titles are taken verbatim, without any modification. The playlist, however, will still have them in the right order. If your player cannot read a playlist, you can also explicitly precede the track name with a number, with leading zeros added automatically again.
